REFERENCE.md

```diff
@@ -36,6 +39,99 @@ Last but not least change your `application.css` to include:
 //= require mtl
 ```
 
+
+
+
+## Extensions to materializecss
```

Comment:
maybe simply call this section CSS extensions and from a purely ordering PoV, shouldn't this be after the layouts section?